BUILD.md

```diff
@@ -175,12 +175,14 @@ Released packages of Linux App can be found in https://repo.nordvpn.com/deb and
 ### Building images
 The following images can be built:
 - builder
+- generator
+- notifier
 - packager
```

Comment:
Probably need add `snaper` to the list as well

Reply:
Added